docs/builder.rst

```diff
@@ -10,10 +10,15 @@ Cookbook
 To create a simple manifest
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-For all manifests, the following fields are *required*.
-    - ``package_name``      (i.e. "owned")
-    - ``version``           (i.e. "1.0.0")
-    - ``manifest_version``  (i.e. "2")
+For all manifests, the following ingredients are *required*.
+
+.. code:: python
+
+   {}
+   package_name(str)
```

Comment:
Maybe since these are all **always** required we should provide a helper for doing them in a single step.

```python
init_manifest(name, version, manifest_version):
    ...
```

Where `manifest_version` isn't required and defaults to the latest version?

Reply:
sounds good, will add in a following pr